.github/workflows/ci.yml

```diff
@@ -65,6 +65,11 @@ jobs:
               pushRemoteCache_$BUILD_KEY \
               -J-Xmx4G
 
+      - name: Run plugin tests
```

Comment:
I don't think scripted tests were running at all - let me know if I missed it somewhere